spec/game_spec.rb

```diff
@@ -0,0 +1,75 @@
+require 'game'
+
+describe Game do
+
+  let(:mock_grid) { ["-", "-", "-", "-", "-", "-", "-", "-", "-"] }
+  let(:second_grid) { [":x", "-", "-", ":x", "-", "-", "-", "-", "-"] }
+  let(:third_grid) { [':o', ':x', ':o', ':x', ':x', ':o', '-', ':o', ':x'] }
+  let(:player1) { double(symbol: :x) }
+  let(:player2) { double(symbol: :o) }
+  let(:board) { double(grid: mock_grid, space_exist?: true, space_is_free?: true, drawing_move?: false, winning_move?: false) }
```

Comment:
Clarity: Could these different boards be set up in blocks with the tests that use them?